hnn_core/network.py

```diff
@@ -1035,6 +1043,35 @@ def clear_connectivity(self):
         """Remove all connections defined in Network.connectivity_list"""
         self.connectivity = list()
 
+    def add_electrode(self, electrode_pos, sigma=3.0, method='psa'):
+        """Specify coordinates of electrodes for LFP recording.
+
+        Parameters
+        ----------
+        electrode_pos : tuple | list of tuple
+            Coordinates specifying the position for LFP electrodes in
+            the form of (x, y, z).
```

Comment:
I think @cjayb is right that we should specify the units here. Should be micrometer?

Reply:
Also `(x, y, z)` is ambiguous: NEURON objects are `(x, z, y)` in my mind (apical dendrite oriented along z)